### ISSUE TITLE
Multiple select title showing line breaks

### ISSUE THREAD

**Issue by kpagcha** [silviomoreto/bootstrap-select#1624]
I am using a multiple select and the `title` attribute of the `select` tag is displaying the data separeated by line breaks.

![1](https://cloud.githubusercontent.com/assets/3865095/22585640/05b9737a-e9f9-11e6-867e-10b4a0a3ab4a.png)

**Comment by aknosis** [comment]
Did you find a resolution to this issue? I am seeing it myself. I'm guessing the issue is that it is taking the <option> content verbatim and therefore including the white space.

e.g.

```html
<option>
    Test
</option>
<option>
    Test 2
</option>
```
vs.
```html
<option>Test</option>
<option>Test2</option>
```

**Comment by caseyjhol** [comment]
Fixed in v1.13.0. See #2063.
